services/venice-controller/src/main/java/com/linkedin/venice/ingestion/control/RealTimeTopicSwitcher.java

```diff
@@ -47,6 +47,7 @@ public class RealTimeTopicSwitcher {
   private final VeniceWriterFactory veniceWriterFactory;
   private final Time timer;
   private final Integer kafkaReplicationFactorForRTTopics;
+  private final Integer kafkaReplicationFactor;
```

Comment:
Can this be `int` instead of `Integer`? Can you update the one above too if that can be changed?

Reply:
done